Data Structures/ReadMe.md

```diff
@@ -1 +1,36 @@
-Data Structures part of the repository
```

Comment:
@shivasinghal2000 Having this README *exactly the same* as the top-level README doesn't make much sense. Please revert this change or alternately you can write 1-2 sentences about data structures 😄

Reply:
@frextrite okay i will do all the required changes